hs_modflow_modelinstance/models.py

```diff
@@ -589,26 +589,13 @@ def metadata(self):
         md = MODFLOWModelInstanceMetaData()
         return self._get_metadata(md)
 
-    def has_required_content_files(self):

```

Comment:
Is this means as long as there is one or more files, the resource can be made public?